features/navbar.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Visitor Can Navigate via Navbar
+    As a visitor,
+    In order to navigate to different sections of the site,
+    I would like to be able to use a navbar
+
+  Scenario: Visitor Sees Navbar Links on Main Page
+    When I visit the site
+    Then I should see "HOME"
```

Comment:
please make sure to refactor the steps here, everything starts with a `Given`
fix the indentation starting at line 6
`And I should see`

Reply:
Fixed